.gitignore

```diff
@@ -0,0 +1,3 @@
+gen/
+target/
+Cargo.lock
```

Comment:
Shouldn't be ignored.
See https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html